extensions/amp-analytics/0.1/vendors.js

```diff
@@ -141,6 +141,35 @@ export const ANALYTICS_CONFIG = /** @type {!JsonObject} */ ({
     },
   },
 
+  'alexametrics': {
+    'requests': {
+      'base': 'https://${ampAtrkHost}/atrk.gif?account=${atrk_acct}&domain=${domain}', 
+      'pageview': '${base}&jsv=amp-${ampVersion}' + 
+        '&frame_height=${viewportHeight}&frame_width=${viewportWidth}' + 
+        '&title=${title}&time=${timestamp}&time_zone_offset=${timezone}' + 
+        '&screen_params=${screenWidth}x${screenHeight}x${screenColorDepth}' + 
+        '&ref_url=${documentReferrer}&host_url=${sourceUrl}&random_number=${random}' + 
+        '&user_cookie=${clientId(__auc)}&user_cookie_flag=0' + 
+        '&user_lang=${browserLanguage}&amp_doc_url=${ampdocUrl}'
+    },
+    'vars': {
+        'atrk_acct': 'xxxxxxxx'
+      , 'domain': 'xxxxxxxx'
+      , 'ampAtrkHost': 'certify-amp.alexametrics.com'
+    },
+    'triggers': {
+      'trackPageview': {
+          'on': 'visible'
+        , 'request': 'pageview'
+      }
+    },
+    'transport' : {
+        'xhrpost': false
```

Comment:
To confirm: Do we only want transport method image here? that server doesn't support the other two method? We recommend using post method if it is available.

Reply:
Yeah. We only want transport method to be image here. POST method is not available on our servers' side.